releasenotes/notes/manylinux2014-e33268fda54e12b1.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    On Linux, the minimum library support has been raised from the
+    `manylinux2010` VM to `manylinux2014` on x86_64.  This mirrors similar
+    changes in Numpy and Scipy.  There should be no meaningful effect for most
+    users, unless your system still contains a very old version of `glibc`.
+    Linux i686 is still built against `manylinux2010`.
```

Comment:
We probably should link to the packaging specs here so people can see what the functional difference is:

manylinux2010: https://www.python.org/dev/peps/pep-0571/
manylinux2014: https://www.python.org/dev/peps/pep-0599/

Reply:
I'll include this message in a PR to the `main` branch, but since we decided not to move to `manylinux2014` for platforms/Pythons we already support, I'm just going to drop this release note from the patch release.